css-pseudo-4/Overview.bs

```diff
@@ -1337,6 +1337,43 @@ details[open]::details-content {
 }</pre>
   </div>
 
+<h3 id="picker-pseudo">
+Styling form control pickers: the ''::picker()'' pseudo-element</h3>
+
+  The <dfn>::picker()</dfn> pseudo-element targets the popup picker of form
+  control elements which have popup pickers, such as the <{select}> element. It
+  is a [=part-like pseudo-element=].
+
+  <pre class=prod>
+    ::picker() == ::picker( <<ident>>+ )
+  </pre>
+
+  The ''::picker()'' pseudo-element only matches when the <a>originating
+  element</a> supports <a>base appearance</a> and has a popup picker. The
+  specified <<ident>> must also match the unique picker name of the
```

Comment:
I think this should probably be defining the term "unique picker name" (for other specs to reference), unless you think it should be defined somewhere else.

Reply:
Yes, I think HTML should define it.